app/src/main/kotlin/at/bitfire/davdroid/ui/setup/GoogleLoginFragment.kt

```diff
@@ -147,13 +148,26 @@ class GoogleLoginFragment(private val defaultEmail: String? = null): Fragment()
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
         val view = ComposeView(requireActivity()).apply {
             setContent {
-                GoogleLogin(defaultEmail = defaultEmail, onLogin = { accountEmail, clientId ->
-                    loginModel.baseURI = googleBaseUri(accountEmail)
-                    loginModel.suggestedAccountName = accountEmail
+                GoogleLogin(defaultEmail = defaultEmail, onLogin = { userEmail, userClientId ->
+                    if (userEmail.isNullOrBlank()) {
+                        Snackbar.make(requireView(), getString(R.string.login_email_address_error), Snackbar.LENGTH_LONG).show()
```

Comment:
I think the Snackbar should rather be an error field of the email input field because this would be more specific. (I tried the PR and the Snackbar was actually not visible because the keyboard was over it).

Reply:
Good thing you noticed. Will change